Source/Core/Common/BitUtils.h

```diff
@@ -201,7 +201,7 @@ inline To BitCast(const From& source) noexcept
   static_assert(std::is_trivially_copyable<To>(),
                 "BitCast destination type must be trivially copyable.");
 
-  std::aligned_storage_t<sizeof(To), alignof(To)> storage;
+  alignas(To) char storage[sizeof(To)];
```

Comment:
Wasn't `std::byte` the type suggested when the deprecation was announced?

Reply:
It was, yeah.